lib/createlocaltracks.js

```diff
@@ -72,7 +72,7 @@ function createLocalTracks(options) {
     audio: options.audio,
     video: options.video
   }).then(function(mediaStream) {
-    var mediaStreamTracks = mediaStream.getTracks();
+    var mediaStreamTracks = mediaStream.getAudioTracks().concat(mediaStream.getVideoTracks());
```

Comment:
It does, but I observed that sometimes video MediaStreamTracks appear before audio MediaStreamTracks. I guess this behavior is not specced, but some of our tests (and maybe even example code we have in the wild) relies on the ability to do

```js
const [audioTrack, videoTrack] = await createLocalTracks()
```